app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -122,5 +119,28 @@ def process_errata_ids(select_all_params)
         []
       end
     end
+
+    private def create_rules
+      rule_clazz = ContentViewFilter.rule_class_for(@filter)
+
+      @rules = []
```

Comment:
Do we need an instance variable here? I'd think create_rules would just return me the rules it created.